src/app/routes/applications/governance-list/governance-list.component.html

```diff
@@ -1,7 +1,6 @@
 <!-- Filter Sub Organization -->
-
 <div class="d-flex align-items-center flex-wrap mb-2" *ngIf="listType === ListType.ORG && orgHierarchy?.length">
-  <div class="d-flex align-items-center my-1" *ngIf="i !== (orgHierarchy?.length - 1)">
```

Comment:
Are you sure this won't break functionality?

Reply:
updated